index.js

```diff
@@ -184,9 +184,9 @@ module.exports = function (content) {
         urls[format] = path.join(pub, url).replace(/\\/g, '/');
 
         if (generatorConfiguration.dest) {
-          self.emitFile(urls[format], res[format]);
+          this.emitFile(urls[format], res[format]);
```

Comment:
I don't think it does because of the big arrow notation I used on line 165, does it?